.github/workflows/ubuntu.yml

```diff
@@ -40,6 +40,9 @@ jobs:
 
       - name: build
         run: cmake --build build -j1
+        
+      - name: test
+        run: cd build/tests/kProcessorLibTests/ && ./testKprocessorLib
```

Comment:
I'd recommend separating this into a different workflow that only works if the build succeeds.

Reply:
And it's currently failing.

Reply:
agree! can you do it?

Reply:
Yes. I will do it later. BTW, what was the difference between the old and new GCC/CMAKE installation commands?